test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/PackageLicenseUtilitiesTests.cs

```diff
@@ -181,5 +245,43 @@ public void PackageLicenseUtility_GenerateCorrectLink()
             Assert.Equal(license, licenseText.Text);
             Assert.Equal("https://licenses.nuget.org/MIT", licenseText.Link.AbsoluteUri);
         }
+
+        private static void CreateDummyPackage(
```

Comment:
This feels like something we already have or if not, should make more globally available to tests. Perhaps a new test fixture.

Reply:
`SimpleTestPathContext` creates a temp directory for tests to write to, but also a nuget.config file that adds a local folder as a package source.  If you look at references to the `PackageSource` property, you can probably find the helper methods that create test packages for Restore tests to use.